ios/sdk/src/callkit/CallKit.m

```diff
@@ -30,7 +30,7 @@
 
 #import <JitsiMeetSDK/JitsiMeetSDK-Swift.h>
 
-#import "../LogUtils.h"
+#import <React/RCTLog.h>
```

Comment:
Can you please move this import together with the other React ones please?

Reply:
resolved!